web/router.go

```diff
@@ -120,8 +120,8 @@ func v2Routes(app *services.ChainlinkApplication, engine *gin.Engine) {
 		authv2.GET("/bridge_types/:BridgeName", bt.Show)
 		authv2.DELETE("/bridge_types/:BridgeName", bt.Destroy)
 
-		w := WithdrawController{app}
-		authv2.POST("/withdraw", w.Create)
+		w := WithdrawalController{app}
+		authv2.POST("/withdrawal", w.Create)
```

Comment:
To be consistent with our existing naming convention, let's make this plural. You can technically create more than one, even if we don't currently have an index endpoint. It might happen one days.

`JobSpecsController` handles `/specs`
`WithdrawalsController` handles `/withdrawals`
`SessionsController` ...